db/migrate/20190916155318_clean_up_course_subject_data_post_migration.rb

```diff
@@ -0,0 +1,60 @@
+# rubocop:disable Metrics/BlockLength
+class CleanUpCourseSubjectDataPostMigration < ActiveRecord::Migration[5.2]
+  def change
+    say_with_time 'cleansing subject data' do
+      courses = RecruitmentCycle.second.courses.includes(:subjects)
```

Comment:
Please lets not do `.second`, we can't guarantee sorting on this.

Reply:
We should hardcode this to the recruitment cycle for "2020" ... that's when we clean up the subjects.